[datadog_org_group] Terraform support for org groups and policies (#3686)
DataDog/terraform-provider-datadog: docs/data-sources/org_group_memberships.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "datadog_org_group_memberships Data Source - terraform-provider-datadog"
subcategory: ""
description: |-

>   Use this data source to retrieve org group memberships filtered by org group or organization. At least one of org_group_id or org_uuid is required.

Reviewer: ```suggestion
Use this data source to retrieve org group memberships. At least one filter (`org_group_id` or `org_uuid`) is required.
```

---

# datadog_org_group_memberships (Data Source)

> Use this data source to retrieve org group memberships filtered by org group or organization. At least one of `org_group_id` or `org_uuid` is required.

Reviewer: ```suggestion
Use this data source to retrieve org group memberships. At least one filter (`org_group_id` or `org_uuid`) is required.
```

## Example Usage

```terraform
# List every org that is a member of the given org group.
data "datadog_org_group_memberships" "by_group" {
  org_group_id = "019d97e8-74d4-7060-806e-e731c8f1d005"
}

# Or look up the group an organization currently belongs to.
data "datadog_org_group_memberships" "by_org" {
  org_uuid = "ff4a8255-6931-58d1-add0-a6b3602d5421"
}
```

<!-- schema generated by tfplugindocs -->
## Schema

### Optional

- `org_group_id` (String) Filter memberships to those within the given org group. At least one of `org_group_id` or `org_uuid` is required. Must be a valid UUID.

> - `org_uuid` (String) Filter memberships to those for the given organization. At least one of `org_group_id` or `org_uuid` is required. Must be a valid UUID.

Reviewer: ```suggestion
- `org_group_id` (String) Filter memberships to those within the given org group. At least one filter (`org_group_id` or `org_uuid`) is required. Must be a valid UUID.
- `org_uuid` (String) Filter memberships to those for the given organization. At least one filter (`org_group_id` or `org_uuid`) is required. Must be a valid UUID.
```

### Read-Only

- `id` (String) The ID of this resource.

> - `memberships` (List of Object) The list of org group memberships. (see [below for nested schema](#nestedatt--memberships))

Reviewer: ```suggestion
- `memberships` (List of Object) The list of org group memberships. See [below for nested schema](#nestedatt--memberships).
```
